cmd/jumphost/cmd.go

```diff
@@ -0,0 +1,148 @@
+package jumphost
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/ec2"
+	"github.com/aws/aws-sdk-go-v2/service/ec2/types"
+	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+	"github.com/openshift/osdctl/pkg/utils"
+	"github.com/spf13/cobra"
+)
+
+const (
+	awsResourceName     = "red-hat-sre-jumphost"
+	publicSubnetTagKey  = "kubernetes.io/role/elb"
+	privateSubnetTagKey = "kubernetes.io/role/internal-elb"
```

Comment:
are publicSubnetTagKey or privateSubnetTagKey used anywhere?

Reply:
It's not used right now, but when I add back the `--cluster-id` flag these tags are commonly used to identify public and private subnets.